### DISCUSSION TITLE
【Web】v4.3.51 => v4.3.68 在load文件时发生了部分问题

### DISCUSSION THREAD

**Discussion by jormungandrE** [Tencent/libpag#2627] [Q&A]
在v4.3.51版本开发中，PC web一切正常，但是iOS的safari【首次进入正常，刷新后一定发生】与部分手机在几次刷新后会提示expected null or instance of _pagLayer, got an instance of _paglayer
为了解决上述问题尝试将libpag.wasm更新至了4.3.57版本，问题如久
再次更新至4.3.68版本则在pagfile.load阶段提示e._malloc is not a function

上述升级是否不太合适，或者我是否在某些阶段使用错了地方，需要提供一份demo吗

**Comment by kindesa** [comment]
这边遇到一些手机，偶尔不播放动画，偶尔可以，一些手机直接卡住主线程，动画也不播放，都是基于chrome浏览器
也是用的v4.3.51版本，升级到4.3.68，提示e._malloc is not a function

**Comment by jinwuwu001** [comment]
请基于release最新版验证下该问题是否还存在，若存在请提供一份demo

**Reply by vivid05** [reply]
最新release web 4.4.31，同样报错

**Comment by vivid05** [comment]
> 在v4.3.51版本开发中，PC web一切正常，但是iOS的safari【首次进入正常，刷新后一定发生】与部分手机在几次刷新后会提示expected null or instance of _pagLayer, got an instance of _paglayer 为了解决上述问题尝试将libpag.wasm更新至了4.3.57版本，问题如久 再次更新至4.3.68版本则在pagfile.load阶段提示e._malloc is not a function
> 
> 上述升级是否不太合适，或者我是否在某些阶段使用错了地方，需要提供一份demo吗

有解决吗

**Reply by jinwuwu001** [reply]
近期我们会发布release/4.5修改该问题

**Comment by jinwuwu001** [comment]
该问题我们近期排查下

**Comment by CodeJhF** [comment]
这条此前我们回复过，之后没有再收到新的反馈。

因长期无回复，且 libpag 4.3 及更早版本已停止维护（当前仅维护 4.4 / 4.5），先关闭本条。
上报版本：4.3.51 / 4.3.68 / 4.3.57。

如果问题在最新版本（4.4 / 4.5）仍能复现，欢迎在本帖回复或新开 issue，并附上 libpag 版本号、平台与系统版本、可复现的 PAG 文件或最小 demo，我们会优先跟进。（stale-close）
